src/Contract/TextEnvelope.purs

```diff
@@ -1,30 +1,15 @@
 module Contract.TextEnvelope
-  ( module TextEnvelope
-  , textEnvelopeBytes
+  ( module X
   ) where
 
-import Contract.Prelude
-
-import Contract.Monad (Contract)
 import Ctl.Internal.Cardano.TextEnvelope
-  ( TextEnvelope(TextEnvelope)
-  , TextEnvelopeDecodeError(JsonDecodeError, CborParseError)
-  , TextEnvelopeType
+  ( TextEnvelopeType
       ( PlutusScriptV1
       , PlutusScriptV2
       , PaymentSigningKeyShelleyed25519
       , StakeSigningKeyShelleyed25519
       )
   , decodeTextEnvelope
-  , printTextEnvelopeDecodeError
-  ) as TextEnvelope
-import Ctl.Internal.Cardano.TextEnvelope (TextEnvelopeType, textEnvelopeBytes) as TE
-import Ctl.Internal.Cardano.TextEnvelope (printTextEnvelopeDecodeError)
-import Ctl.Internal.Types.ByteArray (ByteArray)
-import Data.Bifunctor (lmap)
-import Effect.Exception (error)
-
-textEnvelopeBytes :: String -> TE.TextEnvelopeType -> Contract () ByteArray
-textEnvelopeBytes json ty =
-  liftEither $ lmap (error <<< printTextEnvelopeDecodeError) $
-    TE.textEnvelopeBytes json ty
+  , plutusScriptV1FromEnvelope
+  , plutusScriptV2FromEnvelope
+  ) as X
```

Comment:
`PaymentSigningKeyShelleyed25519` and `StakeSigningKeyShelleyed25519` should have their corresponding functions too.

Reply:
There are such functions in `Ctl.Internal.Wallet.KeyFile` but they work with strings. I think I should make them work with `TextEnvelope`? Maybe reexport here in `Contract.TextEnvelope`?

Reply:
Oh, I seen old branch, I already fixed the types. So the only question is if to reexport them.

Reply:
@klntsky What I should do about it?

Reply:
Thanks, if they accept the envelope type and not string, it's all good.